django_project/dashboard/src/admin_boundaries/Review/DetailMap.tsx

```diff
@@ -245,8 +245,12 @@ export default function ReviewMap(props: ReviewMapInterface) {
         }, 200)
     }, []);
 
+    let _preferences: any = window.preferences
+    // let _maptiler = _preferences ? _preferences['maptiler_api_key'] : ''
```

Comment:
this line is commented out, is it for testing @zamuzakki?

Reply:
Yes, I've reverted the comment.